transpiler/src/parser/core/grammar/BitloopsParser.g4

```diff
@@ -299,7 +299,8 @@ condition
     ;
 
 returnStatement
-    : Return expression?
+    : Return expression?                                   # ReturnSimpleStatement
```

Comment:
I don't love the simple because it is like we have another return as well. maybe we can do it 
#ReturnStatement